### ISSUE TITLE
default uid returns rotating hashes.

### ISSUE THREAD

**Issue by a3626a** [lukeed/uid#9]
With my expectation, uid and uid/single should behaves similarly. Unfortunately default uid returns easily predictable sequences. Buffer Index management could be wrong.

### Expected (uid/single)

each generate hash must be different.

![image](https://github.com/lukeed/uid/assets/4434752/1a293064-4992-4890-8341-5d2a437ad7b7)

### Actual (default uid)

![image](https://github.com/lukeed/uid/assets/4434752/a8459588-19de-459c-9c8a-3972f5dbb36b)

```
676be7876ec
76be7876ec2
6be7876ec20
be7876ec203
e7876ec2037
7876ec20373
876ec20373d
```

It is ROTATING, and next sequence is too easy to predict.

**Comment by StarpTech** [comment]
@lukeed for visibility. People should use the secure variant if possible.

```ts
import { uid } from 'uid/secure';
```

**Comment by lukeed** [comment]
Yes, as designed. UID is just a unique identifier. Readme clearly demarcates uid/secure as being the secure variant, hence the name
